completions/apt-mark

```diff
@@ -0,0 +1,52 @@
+# Debian apt-mark(8) completion                             -*- shell-script -*-
+
+_apt_mark()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+
+    local special i
+    for (( i=0; i < ${#words[@]}-1; i++ )); do
+        if [[ ${words[i]} == @(hold|unhold|auto|manual|install|remove|purge|showhold|showauto|showmanual|showinstall|showremove|showpurge|minimize-manual) ]]; then
```

Comment:
This list would be better sorted correspondingly with the source for easier mainenance/crosscheck. The only source for the full list I'm aware of is the manpage, and at least in the version I have (1.6.12 on Ubuntu 18.04) they're not in this order.

My version doesn't have `minimize-manual`, can you confirm that its inclusion is intentional?